File: selection-sort/selection_sort.py
def selection_sort(arr):
    size = len(arr)
    for i in range(size -1):
        min_index = i
        for j in range(i+1, size):
            if arr[j] < arr[min_index]:
                min_index = j
            
        temp = arr[i]
        arr[i] = arr[min_index]
        arr[min_index] = temp


if __name__ == "__main__":
    print("this is sleecttion sort")
    arr = [64, 25, 12, 22, 11]
    selection_sort(arr)
    print(arr)

File: selection-sort/selection_sort.dart
void selectionSort(List<int> arr) {
  int size = arr.length;
  for (int i = 0; i < size; i++) {
    int min_index = i;
    for (int j = i; j < size - 1; j++) {
      if (arr[min_index] > arr[j]) {
        min_index = j;
      }
    }
    var temp = arr[min_index];
    arr[min_index] = arr[i];
    arr[i] = temp;
  }
}

void main() {
  List<int> arr = [8, 2, 3, 5];
  selectionSort(arr);
  print(arr);
}
